baremaps-openapi/src/main/resources/openapi.yaml

```diff
@@ -1122,7 +1122,6 @@ components:
           type: array
           items:
             type: number
-      additionalProperties: true
```

Comment:
@bchapuis this will break compliance with https://github.com/baremaps/openstreetmap-vecto/blob/main/tileset.json
Anyways we probably rather want to adapt the schema than the code as it introduces the simple-json dependency and makes the code uglier.

Reply:
The tileset openapi schema was updated to support baremaps specific tilesets.